modules/distr-tracing-tempo-config-spanmetrics.adoc

```diff
@@ -96,11 +96,13 @@ spec:
         monitorTab:
           enabled: true # <1>
           prometheusEndpoint: https://thanos-querier.openshift-monitoring.svc.cluster.local:9091 # <2>
+          redMetricsNamespace: "" <3>
         ingress:
           type: route
 ----
 <1> Enables the monitoring tab in the Jaeger console.
 <2> The service name for Thanos Querier from user-workload monitoring.
+<3> Optional: The metrics namespace on which the Jaeger query retrieves the Prometheus metrics. Omit this line if you are using the OpenTelemetry Collector version 0.109.0 or later because these versions use the same default `traces.span.metrics` namespace as the OpenTelemetry Collector uses for this purpose. If you are using an OpenTelemetry Collector version earlier than 0.109.0, add the `redMetricsNamespace: ""` line to configure the correct namespace.
```

Comment:
This was a little confusing to read. However, I think it could probably be said more concisely and clearly as to what to do depending on the version you're running. Making a suggestion here, though it does exclude the piece about why, because I couldn't quite figure out how it fit in well.

```suggestion
<3> Optional: The metrics namespace on which the Jaeger query retrieves the Prometheus metrics. Include this line only if you are using an OpenTelemetry Collector version earlier than 0.109.0. If you are using an OpenTelemetry Collector version 0.109.0 or later, omit this line.
```

Reply:
Thank you for the suggestion, Andrea. Applying.